connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java

```diff
@@ -84,7 +85,9 @@ public Socket afterHandshake() {
     return socket;
   }
 
-  /** Returns {@code true} if running in a Google App Engine Standard runtime. */
+  /**
```

Comment:
Hmm. google-java-format usually prefers single-line Javadoc if it fits. Are you running that?

Reply:
OK. I don't personally use Checkstyle, even with Google settings. What value do you find it has?

Reply:
Checkstyle is enforcement of the style. I spent some time experimenting with using google-java-format and diffing, but checkstyle provides a better explanation of why something is incorrect and is more accessible to Windows users.